e3db/types/note.py

```diff
@@ -0,0 +1,356 @@
+import copy
+from uuid import UUID
+from .note_keys import NoteKeys
+from .note_options import NoteOptions
+
+class Note():
+    def __init__(self, data: dict, note_keys: NoteKeys, note_options: NoteOptions):
+        """
+        Initialize the Note class
+
+        Parameters
+        ----------
+        note_keys : note_keys.NoteKeys
+            Key object that contains encryption, signing, and
+            access keys. 
+
+        note_options : note_options.NoteOptions
+            Options object that contains optional values that
+            are not required to create a note but provide 
+            additional functionality. 
+
+        Returns
+        -------
+        None
+
+        """
+        self.__client_id                = note_options.__client_id
+        self.__mode                     = note_keys.__mode
+        self.__recipient_signing_key    = note_keys.__recipient_signing_key
+        self.__writer_signing_key       = note_keys.__writer_signing_key
+        self.__writer_encryption_key    = note_keys.__writer_encryption_key
+        self.__encrypted_access_key     = note_keys.__encrypted_access_key
+        self.__type                     = note_options.__type
+        self.__data                     = copy.deepcopy(data)
+        self.__plain                    = note_options.__plain
+        self.__file_meta                = note_options.__file_meta
+        self.__max_views                = note_options.__max_views
+        self.__expiration               = note_options.__expiration
+        self.__expires                  = note_options.__expires
+        self.__eacp                     = None 
+
+        self.__note_id                  = None
+        self.__created_at               = None # Is this server defined as per the JS implementation's comment? (see note.js)
```

Comment:
https://github.com/tozny/storage-service/blob/da06d1299c7b4a26967d14082a5cb353efe0ef3b/service/types/note.go#L103

Yep! This is called to set default values to a note